content/blog/2024/funding-czi/index.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Support from CZI to support 2i2c's mission to help communities create and share knowledge with open infrastructure
```

Comment:
I think having "support" twice in this title is a bit redundant. I would perhaps reword.